es/home/javier/Downloads/08_2_Enviando_una_Transaccion_con_Datos.md.txt

```diff
@@ -0,0 +1,127 @@
+# 8.2: Enviando una transacción con datos 
+
+En esta última forma para variar una transacción básica es usar la transacción para enviar datos en lugar de fondos (o en realidad, además de fondos). Esto le brinda la capacidad de incrustar información en la cadena de bloques. Se realiza mediante un comando especial `OP_RETURN`.
+
+El truco? Solo puede almacenar 80 bytes a la vez!
+
+## Crear sus datos.
+
+Lo primero que debe hacer es crear los 80 bytes (o menos) de datos que grabará en su `OP_RETURN`. Esto puede ser tan simple como preparar un mensaje o puede estar usando datos existentes. Por ejemplo, `sha256sum` produce 256 bits de datos, que son 32 bytes, muy por debajo de los límites:
```

Comment:
@javiervargas I think this should be incorporated to translation, because the meaning changes without this. If you have a different idea, please explain it to me below.